src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_NUMBER = 5;
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    
+    public Figure getRandomFigure() {
+        int figureIndex = random.nextInt(FIGURES_NUMBER);
+        switch (figureIndex) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomIsoscelesTrapezoid();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomRightTriangle();
+            case 4:
+                return getRandomSquare();
+            default:
+                return new Circle(colorSupplier.getRandomColor(), getRandomInt());
```

Comment:
```suggestion
                return getDefaultFigure();
```

I think it's better to do this way